app/views/full-page-examples/what-is-your-postcode/index.njk

```diff
@@ -46,7 +46,8 @@ scenario: >-
       id: "address-postcode",
       name: "address-postcode",
       autocomplete: "postal-code",
-      errorMessage: errors["address-postcode"]
+      errorMessage: errors["address-postcode"],
+      autocomplete: "postcode"
```

Comment:
I don't think this is needed because on line 48 there is already an autocomplete declaration.